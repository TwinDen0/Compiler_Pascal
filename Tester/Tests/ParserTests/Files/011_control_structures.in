program CheckFor;
var
i : integer;
begin
for i := 5 downto 6 do write(i);
end.