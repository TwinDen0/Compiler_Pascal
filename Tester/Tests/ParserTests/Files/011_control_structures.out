CheckFor
├─── var
│    └─── integer
│         └─── i
├─── begin
│    └─── for 
│         ├─── :=
│         │    ├─── i
│         │    └─── 5
│         ├─── downto
│         │    └─── 6
│         └─── write
│              └─── i
└─── end


